applications_tests/gls_navier_stokes_2d/high_pe_stab_gls.output

```diff
@@ -7,7 +7,7 @@ Running on 1 MPI rank(s)...
 *****************************************************************
 Steady iteration :        1/3
 *****************************************************************
-L2 error velocity : 0
+L2 error velocity : 5.55112e-17
```

Comment:
I have an impression that this might make the test very fragile...  Maybe we can add a switch or something to prevent velocity error calculation of something if the test breaks?